newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -0,0 +1,835 @@
+        );
+      }
+    },
+    [currentLeaderboard, setDisplayOnlyBestEntry]
+  );
+
+  const onCopy = React.useCallback(
+    () => {
+      if (!currentLeaderboard) return;
+      // TODO: use Clipboard.js, after it's been reworked to use this API and handle text.
+      navigator.clipboard.writeText(currentLeaderboard.id);
+    },
+    [currentLeaderboard]
+  );
+  if (!isOnline) {
+    return (
+      <WrappedError>
+        <PlaceholderError>
+          <Trans>
+            An internet connection is required to administrate your game's
+            leaderboards.
+          </Trans>
+        </PlaceholderError>
+      </WrappedError>
+    );
+  }
+  if (apiError && apiError.action === 'leaderboardCreation') {
+    return (
+      <WrappedError>
+        <AlertMessage kind="error">{apiError.message}</AlertMessage>
+      </WrappedError>
+    );
+  }
+  if (apiError && apiError.action === 'leaderboardsFetching') {
+    return (
+      <WrappedError>
+        <PlaceholderError onRetry={_listLeaderboards} kind="error">
+          {apiError.message}
+        </PlaceholderError>
+      </WrappedError>
+    );
+  }
+  if (leaderboards === null) {
+    if (isRequestPending) return <PlaceholderLoader />;
+    else {
+      return (
+        <WrappedError>
+          <PlaceholderError onRetry={_listLeaderboards} kind="error">
+            <Trans>
+              An error ocurred when retrieving leaderboards, please try again
+              later.
+            </Trans>
+          </PlaceholderError>
+        </WrappedError>
+      );
+    }
+  }
+
+  if (!!leaderboards && leaderboards.length === 0)
+    return (
+      <Line noMargin expand justifyContent="center" alignItems="center">
+        <EmptyPlaceholder
+          title={<Trans>Create your game's first leaderboard</Trans>}
+          description={<Trans>Leaderboards help retain your players</Trans>}
+          actionLabel={<Trans>Create a leaderboard</Trans>}
+          onAdd={() => {
+            _createLeaderboard();
+          }}
+          isLoading={isRequestPending}
+        />
+      </Line>
+    );
+
+  const leaderboardDescription = (
+    i18n: I18nType,
+    currentLeaderboard: Leaderboard
+  ) => [
+    {
+      key: 'name',
+      avatar: <Label />,
+      text: isEditingName ? (
+        <Line alignItems="center">
+          <TextField
+            ref={newNameTextFieldRef}
+            margin="none"
+            fullWidth
+            maxLength={50}
+            value={newName}
+            errorText={newNameError}
+            onChange={(e, text) => setNewName(text)}
+            onKeyPress={event => {
+              if (event.key === 'Enter' && !isRequestPending) {
+                _updateLeaderboard(i18n, { name: newName });
+              }
+            }}
+            disabled={isRequestPending}
+          />
+          {!isRequestPending && (
+            <IconButton
+              style={{ padding: 0, marginLeft: 5 }}
```

Comment:
See if you can add a `<Spacer />` instead of marginLeft: 5.

`Spacer` helps to keep the margin consistent in the whole app.

Reply:
Yes true, I'll change that